collectionApp_2020-02-04.sql

```diff
@@ -0,0 +1,80 @@
+# ************************************************************
```

Comment:
delete the old database (or move it somewhere outside of the folders git is checking)

Reply:
thanks deleted